crates/deno_task_shell/Cargo.toml

```diff
@@ -24,10 +24,11 @@ tokio-util = { version = "0.7.12", optional = true }
 os_pipe = { version = "1.2.1", optional = true }
 serde = { version = "1", features = ["derive"], optional = true }
 thiserror = "1.0.63"
-pest = "2.7.12"
+pest = { git = "https://github.com/pest-parser/pest.git", branch = "master", features = ["miette-error"] }
 pest_derive = "2.7.12"
 dirs = "5.0.1"
-pest_ascii_tree = "0.1.0"
+pest_ascii_tree = { git = "https://github.com/prsabahrami/pest_ascii_tree.git", branch = "master" }
```

Comment:
Why do you need this custom git repository instead of the released package?

Reply:
Its because pest_ascii_tree uses the latest released version of pest. But, pest didn't release a new version after miette support. Hence, having the two causes conflicts since there are differences in struct definitions between the git repo version of pest and the released. 
```rs
error[E0308]: mismatched types
   --> crates/deno_task_shell/src/parser.rs:370:37
    |
370 |   pest_ascii_tree::print_ascii_tree(parsed);
    |   --------------------------------- ^^^^^^ expected `Result<Pairs<'_, _>, Error<_>>`, found `Result<Pairs<'_, Rule>, Error<...>>`
    |   |
    |   arguments to this function are incorrect
    |
    = note: `Pairs<'_, Rule>` and `Pairs<'_, _>` have similar names, but are actually distinct types
note: `Pairs<'_, Rule>` is defined in crate `pest`
   --> /Users/parsabahraminejad/.cargo/git/checkouts/pest-25f0f91498b61172/65e5b2b/pest/src/iterators/pairs.rs:36:1
    |
36  | pub struct Pairs<'i, R> {
    | ^^^^^^^^^^^^^^^^^^^^^^^
note: `Pairs<'_, _>` is defined in crate `pest`
   --> /Users/parsabahraminejad/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pest-2.7.12/src/iterators/pairs.rs:36:1
    |
36  | pub struct Pairs<'i, R> {
    | ^^^^^^^^^^^^^^^^^^^^^^^
    = note: perhaps two different versions of crate `pest` are being used?
note: function defined here
   --> /Users/parsabahraminejad/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pest_ascii_tree-0.1.0/src/lib.rs:170:8
    |
170 | pub fn print_ascii_tree<R>(parsing_result: Result<Pairs<R>, Error<R>>)
    |        ^^^^^^^^^^^^^^^^ 
```
Though I might be wrong to fix this with updating the dependency. :))

Reply:
I see, excellent.